lambdas/service/app.py

```diff
@@ -860,69 +909,39 @@ def start_manifest_generation():
                     status_code=301 if wait_time else 302)
 
 
-@app.route('/fetch/manifest/files', methods=['GET'], cors=True)
+@app.route('/fetch/manifest/files', methods=['GET'], cors=True, path_spec=manifest_path_spec(), method_spec={
+    'description': format_description('''
+        Initiate and check status of a manifest generation job, returning a
+        200 response with simulated headers in the body. If the client receives
+        a response body with the `Status` field set to 301, the client should
+        wait the number of seconds specified in `Retry-After` and then request
+        the URL given in the `Location` field. The URL will point back at this
+        endpoint so the client should expect a response of the same shape.
+
+        Note that the actual HTTP response is of status 200, only the `Status`
+        field of the body will be 301. The intent is to emulate HTTP while
+        bypassing the default client behavior which, in most web browsers, is
+        to ignore `Retry-After`. The response described here is intended to be
+        processed by client-side Javascript such that the recommended delay in
+        `Retry-After` can be handled in Javascript rather than relying on the
+        native implementation by the web browser.
+
+        The client should request the URL given in the `Location` field. The
+        URL will point to a different service and the client should expect a
+        response containing the actual manifest. Currently the `Location` field
+        of the final response is a signed URL to an object in S3 but clients
+        should not depend on that.
+    '''),
+    'responses': {
+        '200': {
+            **manifest_response_content(include_status=True),
+            'description': 'The manifest generation has been started or is still ongoing',
+        }
+    },
+    'summary': 'Describe current version of the Azul service',
```

Comment:
Better to write from hand instead of copying and pasting. Summary should precede description.

Reply:
The swagger specification didn't specify a preferred ordering of keys so I thought alphabetical would be a consistent scheme for us to use. It does unfortunately result in `description` key being separated from the `summary` key though. I will try to re-order the keys based on a more contextual ordering.